pyproject.toml

```
@@ -42,6 +42,7 @@ tune = "torchtune._cli.tune:main"
 [project.optional-dependencies]
 dev = [
     "bitsandbytes>=0.43.0",
+    "comet_ml",
```

```suggestion
    "comet_ml>=3.44.2",
```
This version contains our latest change to our new way of initializing Comet Experiment